tests/bats/deploy/govcms-db-update.bats

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bats
+# shellcheck disable=SC2002,SC2031,SC2030
```

Comment:
just updating this test to make it consistent with others:
- added shellcheck bypass rules
- moved an assertion of the number of drush calls to the bottom of each test. We want this assertion to always be the last to make sure that it is "visible" and is always up-to-date.